department-of-veterans-affairs/vets-website
# Pension 69752 [MVP] Step 3: special monthly pension status

## Summary

On Step 3, Health and employment information,
create new page for Special monthly pension,
as shown on the designs.

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Select 'Start your application without signing in'
3. Proceed to Chapter 3 by filling out the required fields on Chapter 1 and Chapter 2
4. Verify the fourth page in Chapter 3, Special Monthly Pension,  matches acceptance criteria

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69752) 
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done
* [x]  Form page manually accessed and yes, no, and null answers tested

## Screenshots



|         | Before | After ('No') | After ('Yes') | After (validation error) |
| ---- | ------- | ----------- | ------------ | --------------------- |
| Mobile  |  New page  |    <img width="330" alt="no-mobile" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/b5a6b193-9625-4c58-a619-69ce297e9bad">   | <img width="325" alt="yes-mobile" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/e1ec5270-6712-49fa-8758-0d4f91377ad0"> | <img width="326" alt="error-mobile" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/dfc08946-06a2-447c-bf9a-9bff0d74f3a6"> |
| Desktop |   New Page    |    <img width="544" alt="no-desktop" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/bec5e424-e090-4824-9593-910284423550">| <img width="531" alt="yes-desktop" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/6852ce95-28e7-4f35-82e2-5a85bba6a09b"> | <img width="528" alt="error-desktop" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/43cadf25-e896-490c-86a1-bbcb506958d7"> |


## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69752) 

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:
